### ISSUE TITLE
i think wrong deprecation warnings for regions/regions_attribute have been added

### ISSUE THREAD

**Issue by loeken** [ovh/terraform-provider-ovh#228]
Hi there!

really appreciate the work on all parties involved in the https://github.com/ovh/terraform-provider-ovh/pull/198 to allow us using a private_network_id when creating a new kuberntes cluster, however i think a small mistake was made with the deprecation messages.

### Terraform Version
1.0.8

### Affected Resource(s)
- ovh_cloud_project_network_private

### Terraform Configuration Files
```hcl
resource "ovh_cloud_project_network_private" "net" {
   service_name = var.service_name
   name       = var.kubernetes_cluster_name
   vlan_id    = var.kubernetes_vlan_id
   regions = [var.os_region_name]
}

```

### Debug Output
terraform plan results in these warnings:
╷
│ Warning: Attribute is deprecated
│ 
│   with ovh_cloud_project_network_private.net,
│   on main.tf line 37, in resource "ovh_cloud_project_network_private" "net":
│   37:    regions = [var.os_region_name]
│ 
│ use the regions_attributes field instead
╵
╷
│ Warning: Deprecated Attribute
│ 
│   with ovh_cloud_project_network_private.net,
│   on main.tf line 37, in resource "ovh_cloud_project_network_private" "net":
│   37:    regions = [var.os_region_name]
│ 
│ use the regions_attributes field instead

### Note

I also looked at the documentation for 0.16 https://registry.terraform.io/providers/ovh/ovh/latest/docs/resources/cloud_project_network_private and expected regions_attributes to be listed in the "Attributes Reference" section so i think the fields got mixed up along the way

**Comment by lyrixx** [comment]
I have the same issue ✋🏼 
We are using:
```
terraform {
  required_providers {
    ovh = {
      source  = "ovh/ovh"
      version = "0.16.0"
    }
```

**Comment by scraly** [comment]
Hi,

thanks for the issue.
For your information, we already fixed the wrong deprecation message and updated the documentation as well:
https://github.com/ovh/terraform-provider-ovh/pull/238

A new version of the Terraform provider should be released in the beginning of 2022 :) 

Thanks

**Comment by scraly** [comment]
A new release have been published.
The fix have been released in the version 0.17.0:
https://registry.terraform.io/providers/ovh/ovh/0.17.0
